web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html

```diff
@@ -1,25 +1,23 @@
-<div layout-padding layout="column" class="map-page">
-
-  <div *ngIf="!targetEntity" class="map-container">
-    <em>Before configuring this mapping you must set a target entity. You can create new entities in the <a [routerLink]="['/entities']">Entities</a> view.</em>
-  </div>
-
-  <div *ngIf="targetEntity && !targetEntity.hasDocs" class="map-container">
-    <em>Before configuring this mapping you must ingest source documents.</em>
-  </div>
-
-  <div [ngClass]="{ hidden: !(targetEntity && targetEntity.hasDocs)}" class="map-container">
-
-    <div id="source">
-      <div id="source-heading">
-        <div class="item-title">
-          <div class="item-type">Source</div>
-          <span class="help-icon">
-            <a href="https://marklogic.github.io/marklogic-data-hub/harmonize/mapping/#changing-the-mapping-source-document" target="_blank"><i class="fa fa-question-circle fa-lg"></i></a>
-          </span>
-        </div>
-        <p *ngIf="!editingURI" class="item-identifying-info">
-          <span class="uri-label" tooltip="The URI of the source document from which QuickStart generates the list of source property names." container="body">URI:</span>
+<!-- layout-padding -->
+<!-- New Source Design for mapping expression -->
+
+<div class="source-prop-container-map-exp">
+    
+    <span>
+  
+   <div class="Source_Tbl">
+      <div class="item-title">
+      <div class="item-type">Source Data</div>
+      <span class="help-icon">
+          <a href="https://marklogic.github.io/marklogic-data-hub/harmonize/mapping/#changing-the-mapping-source-document" target="_blank"><i class="fa fa-question-circle fa-lg"></i></a>
+        </span>
+      </div>
+      <p class="item-identifying-info">
+          <span class="uri-label" tooltip="The Query of the source document from which QuickStart generates the list of source property names." container="body">Query: </span>
+          <span class="edit-source-query" (click)="this.OpenFullSourceQuery()" tooltip="{{ this.step.options.sourceQuery }}" container="body">{{ getInitialChars(this.step.options.sourceQuery, 30, '...') }}</span>
+      </p>
+      <p *ngIf="!editingURI" class="item-identifying-info">
+          <span class="uri-label" tooltip="The URI of the source document from which QuickStart generates the list of source property names." container="body">URI: </span>
```

Comment:
Couple UI thoughts here:

- When I edit the URI, the Source Data column expands. I think what it expands to is a better size than its default size, as everything in the Source Data table is pretty compacted and there's a lot of wrapping of values.
- The two tables aren't aligned - the Entity mapping table is lined up with "URI:" instead of with the top of the Source Data table. The mockup shows them aligned, so I think that should be fixed.